src/sparseml/pytorch/optim/manager.py

```diff
@@ -398,6 +399,11 @@ def initialize(
         :param kwargs: Optional kwargs to support specific arguments
             for individual modifiers.
         """
+        if (python_log and loggers) and not any(
+            isinstance(logger, PythonLogger) for logger in loggers
```

Comment:
`not any(isinstance(logger, PythonLogger) for logger in loggers)` will evaluate to False if at least one logger is  a `PythonLogger` object. So the goal is to assert that we always have at least one `PythonLogger` in `loggers` right?